Cargo.toml

```diff
@@ -10,7 +10,7 @@ name = "flatmanager"
 path = "src/lib.rs"
 
 [dependencies]
-env_logger = "0.5"
+env_logger = "0.8.3"
```

Comment:
Do we have to enforce patch releases as well? `0.8` should be fine

Reply:
true, this was the making of `cargo upgrade`